### ISSUE TITLE
[Survival]-24w30a Survival Mode Misc. Bug List

### ISSUE THREAD

**Issue by Crimson-Artist** [tsunamayo/Starship-EVO#5899]
After trying the survival mode again I've found several bugs.

1. Ore and stone have a have a number attached to each one. probably an oversight. Large stone also can not be used to make stone blocks at the crafting bench despite having them my inventory. 
![20240721144900_1](https://github.com/user-attachments/assets/4f4486a6-44bb-419d-8cdc-3db13db3009e)
2. Tools when placed on on the hotbar for some reason take up inventory space. I have the mining tool on my hotbar and it is taking up 40 units. Every other tool has different weights. 
![20240721153414_1](https://github.com/user-attachments/assets/21c001ba-69ee-4f00-8d9c-81f89b88361c)
3. The unsaved entity tool tip keeps coming up and it gets in the way. 
![20240721174912_1](https://github.com/user-attachments/assets/ddb35832-c9b6-4696-ba03-53f759744ea1)
4. Collectors transferring items into a cargo box is extremely slow and prevents new items from being sucked in.
5. Ships that receive the void pearls in cargo warning are prevented from warping out even when all pearls have been removed from cargo and placed in void pearl storage. This persisted until I logged out and was fixed upon login back in.

**Comment by tsunamayo** [comment]
Issue 1) fixed.

**Comment by tsunamayo** [comment]
Issue 2) fixed.

**Comment by tsunamayo** [comment]
For 3) now  the message only appears in creative.

**Comment by tsunamayo** [comment]
For 4) I would need more info, can you send me your setup? The bigger the collector / pipe, the faster it will transmit.

**Comment by Crimson-Artist** [comment]
My ship has both a 2m collector and a 2m pipe. The transfer speed is at the end of the video. I also caught another bug on the video here. freshly mined ore does wont go in to the collector at all. after ive gathered up the ore and tossed it on the ground only then does the collector want to suck them up.
https://github.com/user-attachments/assets/90302556-1279-44cc-be83-404b0aa0f7c7

**Comment by tsunamayo** [comment]
@Crimson-Artist Can you send me your build? I see mass mining with big beam will cause some trouble. I think I will need to lower the amount of metal spawned.

**Comment by tsunamayo** [comment]
Issue 5) fixed. 6) has been booked elsewhere.

**Comment by Crimson-Artist** [comment]
This is the ship I used.
[Survival ship.zip](https://github.com/user-attachments/files/16329414/Survival.ship.zip)

**Comment by Crimson-Artist** [comment]
Theres also several creative mode abilities that can be turned on in survival such as F5 no clip mode as well as some of the F1 menu toggles like the fast move keys and teleport to any entity option.

Maybe for survival mode you can add a unstuck button like several other games to replace the no clip mode. There have been several instances where I got stuck inside of my ship cuz I used the jump pack to try and get on the roof only to boost into it and get stuck. I had to remove the block in order to unstick myself.

**Comment by tsunamayo** [comment]
> Theres also several creative mode abilities that can be turned on in survival such as F5 no clip mode as well as some of the F1 menu toggles like the fast move keys and teleport to any entity option.
> 
> Maybe for survival mode you can add a unstuck button like several other games to replace the no clip mode. There have been several instances where I got stuck inside of my ship cuz I used the jump pack to try and get on the roof only to boost into it and get stuck. I had to remove the block in order to unstick myself.

Yes  there are there as people where still having too much trouble at first, but I plan to remove them when ready. How the unstuck mode is supposed to be different from the F5?

**Comment by Crimson-Artist** [comment]
An unstuck button usually causes the player to teleport a few feet away from their current position or the next free available space. usually thats enough to get the player unstuck from a piece of geometry. Most other survival games have this button in the esc menu since its not suppose to be a part of normal game play.
